### DISCUSSION TITLE
Polarization vector with outgoing momenta and DoPolarizationSum[]

### DISCUSSION THREAD

**Discussion by manuel-morales-a** [FeynCalc/feyncalc#136] [Q&A]
Hi everyone, 

I have a diagram with one outgoing gluon. In the amplitude, therefore, I use a Conjugate[PolarizationVector[k, mu]].

I understand from the documentation that the momentum in the PolarizationVector[k, mu] is "incoming", so if the momentum of the gluon is outgoing , in my amplitude I just use Conjugate[PolarizationVector[-k, mu]]. However, when I do DoPolarizationSums[#, -k, n], I see that there are still polarization tensors that have not been reduced (they are contracted with other four-momenta of the diagram, and with itself). What am I doing wrong?

Thanks in advance. Best!

**Comment by vsht** [comment]
Hi,

the symbol denoting the 4-momentum in `PolarizationVector` has no direction attached to it.
Whether it is outgoing or incoming is determined solely by the polarization vector being
conjugated or not. It is the exactly the same notation as in the textbooks:

- $\varepsilon^\mu(k)$ -> `PolarizationVector[k, mu]`
- $\varepsilon^{\ast \mu} (k)$ -> `ComplexConjugate[PolarizationVector[k, mu]]`

This is why the second argument of `DoPolarizationSums` must be a momentum, 
not "minus momentum".

>I understand from the documentation that the momentum in the PolarizationVector[k, mu] is "incoming", so if the momentum of the gluon is outgoing , in my amplitude I just use Conjugate[PolarizationVector[-k, mu]]. 

Where exactly does the [documentation](https://feyncalc.github.io/referenceDev) imply that one needs to 
switch the sign to get the outgoing polarization vector? If it is so, I should correct the relevant statements
to avoid confusion.

Cheers,
Vladyslav

**Comment by manuel-morales-a** [comment]
Hi Vladyslav, 

Thank you for your answer!

I have been using [this reference](https://feyncalc.github.io/FeynCalcBook/ref/Polarization.html), as I was writing the PolarizationVector[] in StandardForm. The explicit "(incoming)" in the definition made me think that the direction was an independent parameter that had to be specified.

By the way, the details of [GluonVertex[]](https://feyncalc.github.io/reference) also specify that the momenta are flowing into the vertex. If, to this vertex, I attach two conjugate polarization vectors (say, for example, to calculate the amplitude of g -> gg), does one have to change the signs of their momenta (k -> -k) in the vertex function, or the fact that the momenta also appear inside ComplexConjugate[PolarizationVector[]] alredy gives their direction?

Best wishes, 

Manuel

**Reply by vsht** [reply]
> I have been using [this reference](https://feyncalc.github.io/FeynCalcBook/ref/Polarization.html), as I was writing the PolarizationVector[] in StandardForm. The explicit "(incoming)" in the definition made me think that the direction was an independent parameter that had to be specified.

Thanks for the pointer. Now that I read it, the sentence on its own makes sense, since `Polarization[k]` (which becomes `Polarization[k,I]`) indeed denotes an incoming vector, while `Polarization[k,-I]` means an outgoing one. But I'll try to reformulate it in a better way to avoid possible confusion.

> By the way, the details of GluonVertex[] also specify that the momenta are flowing into the vertex. If, to this vertex, I attach two conjugate polarization vectors (say, for example, to calculate the amplitude of g -> gg), does one have to change the signs of their momenta (k -> -k) in the vertex function, or the fact that the momenta also appear inside ComplexConjugate[PolarizationVector[]] alredy gives their direction?

I actually never use GluonVertex[], but rather employ FeynArts/FeynRules/QGRAF for all my diagrammatic needs. Everything else is too error prone IMHO. But in general it is again like writing Feynman diagrams with the rules from a book (where the momenta are usually all incoming): you need to switch the directions of momenta in GluonVertex[] by hand to match the momentum flow in the diagram. Since you are basically multiplying different building block with each other, there is no way GluonVertex[] can know anything about  neighboring polarization vertices.

Cheers,
Vladyslav

**Reply by vsht** [reply]
BTW, the current documentation for the dev version is here: https://feyncalc.github.io/referenceDev
